design/protocol-changes/protocol-parameters.tex

```diff
@@ -0,0 +1,48 @@
+\section{Protocol parameters}\label{sec:protocol parameters}
+
+The Peras protocol is influenced by various new protocol parameters, also see~\cite{peras-cip}, which we list in \cref{fig:protocol parameters}.
+For explicitness and to avoid confusion, we use a relatively verbose naming scheme.
+The Cardano Ledger allows protocol parameters to be controlled via on-chain governance.
+We indicate whether this flexibility is likely to be useful.
+
+\begin{figure}[h]
+  \centering
+  \begin{tabular}{c c c c c}
+    \toprule
+    Name & Unit & Symbol & Feasible value & Governable? \\
+    \midrule
+    \perasRoundSlots{} & \unit{\slot} & $U$ & \num{90} & $\checkmark$ \\
+    \perasBlockMinSlots{} & \unit{\slot} & $L$ & $T_{\mathrm{CQ}} = \kcg/\asc$ & $\checkmark$ \\
+    \perasBlockMaxSlots{} & \unit{\slot} & n.a. & \numrange{30}{90} & $\checkmark$ \\
+    \perasCertMaxSlots{} & \unit{\slot} & $A$ & \alex[inline,inlinewidth=4cm]{see ongoing Peras CIP discussion} & ✘ \\
+    \perasIgnoranceRounds{} & \unit{\slot} & $R$ & \alex[inline,inlinewidth=4cm]{see ongoing Peras CIP discussion} & ✘ \\
+    \perasCooldownRounds{} & \unit{\slot} & $K$ & \alex[inline,inlinewidth=4cm]{see ongoing Peras CIP discussion} & ✘ \\
+    \perasBoost{} & blocks & $B$ & \num{15} & $\checkmark$ \\
+    \perasQuorum{} & weight & $\tau$ & $3/4 \cdot \perasN$ & ✘ \\
+    \perasN{} & committee seats & $N$ & \numrange{500}{1000} & $\checkmark$ \\
+    \perasVoteSizeLimit{} & \unit{\byte} & n.a. & \qty{200}{\byte} & $\checkmark$ \\
+    \perasCertSizeLimit{} & \unit{\byte} & n.a. & \qty{20}{\kilo\byte} & $\checkmark$ \\
+    \bottomrule
+  \end{tabular}
+  \caption{Peras protocol parameters}\label{fig:protocol parameters}
+\end{figure}
```

Comment:
Do we expect people to go look for the definition of the symbols elsewhere, eg. [Bai+25], or should we include a description of what the symbols mean? @nbacquey and I think it would be worth describing the symbols here.

Reply:
Yeah, adding short descriptions sounds good. In general, I don't want to duplicate too much material from existing documents, but this sounds fine/desirable.